android/app/src/main/java/org/fossasia/openevent/viewmodels/auth/SignUpActivityViewModel.java

```diff
@@ -0,0 +1,60 @@
+package org.fossasia.openevent.viewmodels.auth;
+
+import android.arch.lifecycle.LiveData;
+import android.arch.lifecycle.MutableLiveData;
+import android.arch.lifecycle.ViewModel;
+
```

Comment:
remove this space please

Reply:
This space is automatically provided by Android studio. Android studio automatically groups the similar imports as one and adds spaces between unrelated imports .It's suitable not to change it.